### ISSUE TITLE
Refreshing logs you out again

### ISSUE THREAD

**Issue by eavesr** [davedean/tdcm#20]

**Comment by davedean** [comment]
yep, I hate it. If you comment out the USER env var in docker-compose, it'll disable auth for now.

I'll fix the basic auth or move to something better soon.
